pipeline/src/verify.rs

```diff
@@ -34,7 +34,7 @@ pub fn verify(temp_dir: &Path, name: &str, constants_name: Option<&str>) -> Resu
         return Err("Pil verifier run was unsuccessful.".to_string());
     } else {
         let output = String::from_utf8(verifier_output.stdout).unwrap();
-        log::error!("PIL verifier output: {}", output);
+        log::info!("PIL verifier output: {}", output);
```

Comment:
Maybe this should even be debug?

Reply:
Or maybe we should only log on error? It kind of clutters the output a lot.

Reply:
probably